### PR TITLE
XSOAR EDL Checker - Add support for XSOAR 8.x - Oct 2023

### PR DESCRIPTION
**Pull request by beauchompers**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N/A

## Description
Added support to check EDLs hosted on XSOAR 8.x, and fixed an issue with the playbook where the conditional was looking at the wrong output to determine if any EDLs were failing. 

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Checks an XSOAR EDL to make sure it's returning a valid response.
-This integration was integrated and tested with version 6.8+ of Cortex XSOAR.
+Checks an XSOAR hosted EDL to make sure it's returning a valid response.   Supports PAN-OS (text), CSV, or JSON EDLs.
+This integration was integrated and tested with version xx of XSOAR EDL Checker.
```
Comment (MosheEichler) [inline Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md:2]:
```suggestion
This integration was integrated and tested with version ????? of XSOAR EDL Checker.
```
Please replace with the correct version.

**Review by MosheEichler — COMMENTED** [review]
Hi @beauchompers,
Thank you for your contribution!
Good work :)

Plesae see my comments.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-Checks an XSOAR EDL to make sure it's returning a valid response.
-This integration was integrated and tested with version 6.8+ of Cortex XSOAR.
+Checks an XSOAR hosted EDL to make sure it's returning a valid response.   Supports PAN-OS (text), CSV, or JSON EDLs.
```
Comment (MosheEichler) [inline Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md:1]:
```suggestion
Checks an XSOAR hosted EDL to make sure it's returning a valid response. Supports PAN-OS (text), CSV, or JSON EDLs.
```

File: Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/XSOAREDLChecker.py [hunk 3]
```diff
@@ -11,12 +10,26 @@
...
+    """
+    Returns the url to be used to check the EDL, depends on the XSOAR version.
+    """
+    url = demisto.demistoUrls().get('server')
+    if xsoar_version == "6.x":
+        # return the server url for xsoar 6
+        return url
+    else:
+        # construct the url for xsoar 8
+        url = f"{url[:8]}ext-{url[8:]}/xsoar"
```
[the remark this answers is not in the document]
Comment (beauchompers) [reply Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/XSOAREDLChecker.py:23]: Long running integrations operate on a different domain and path in XSOAR 8.  This simply constructs the base_url to match that. 

https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Forward-Requests-to-Long-Running-Integrations

File: Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md (hunk 1 above)
Comment (beauchompers) [reply Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md:2]: As this integration works with Cortex XSOAR and not an external system, updated to say which versions of XSOAR it was tested with (6.12 and 8.4) as well as the version of the Generic Export Indicators Service that provides the EDLs.

File: Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md (hunk 2 above)
Comment (beauchompers) [reply Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md:1]: Fixed the spaces.

**Review by MosheEichler — APPROVED** [review]

**Pull request merged**